backend/tests/browser/smoke_server.py

```
@@ -0,0 +1,105 @@
+from pathlib import Path
+
+BACKEND_ROOT = Path(__file__).resolve().parents[2]
+sys.path.insert(0, str(BACKEND_ROOT))
+
+os.environ.setdefault("ENVIRONMENT", "test")
+os.environ.setdefault("AUTH_DISABLED", "true")
+os.environ.setdefault("ALLOW_AUTH_DISABLED_IN_PRODUCTION", "true")
+os.environ.setdefault("SECRET_KEY", "browser-smoke-secret-key-change-me")
+os.environ.setdefault(
+    "DATABASE_URL",
+    f"sqlite:///{Path('/tmp/dmarq-browser-smoke.sqlite').as_posix()}",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/dmarq-browser-smoke.sqlite"

(S108)